test/unit/firmata.test.js

```diff
@@ -890,66 +877,59 @@ describe("Board: lifecycle", function() {
   });
 
   it("forwards 'close' events from transport", function(done) {
-    var transport = new SerialPort("/path/to/fake/usb");
+    var transport = new SerialPort(port);
     var board = new Board(transport, initNoop);
-
+    board.on("error", done);
     board.on("close", done);
 
     transport.emit("close");
   });
 
-  it("forwards 'disconnect' events from transport", function(done) {
-    var transport = new SerialPort("/path/to/fake/usb");
-    var board = new Board(transport, initNoop);
-
-    board.on("disconnect", done);
-
-    transport.emit("disconnect");
-  });
-
   it("forwards 'error' event from transport", function(done) {
-    var transport = new SerialPort("/path/to/fake/usb");
+    var transport = new SerialPort(port);
     var board = new Board(transport, initNoop);
-
-    board.on("error", done);
+    const error = new Error('I am a teapot');
+    board.on("error", function(err) {
+      assert.ok(err === error);
```

Comment:
Whenever possible, I try to get rid of `ok(...)` in favor of `assert.equal(actual, expected)`